src/js/view/controls/controls.js

```diff
@@ -1,31 +1,386 @@
+            this.rightClickMenu = null;
+            this.showing = false;
+            this.unmuteCallback = null;
 
             const element = this.context.createElement('div');
             element.className = 'jw-controls jw-reset';
             this.element = element;
 
-            this.enable();
+            const right = document.createElement('div');
+            right.className = 'jw-controls-right jw-reset';
+            element.appendChild(right);
+            this.right = right;
         }
 
-        enable() {
+        enable(api, model, videoLayer) {
+            // Dock Area and Buttons
+            if (!this.dock) {
+                this.dock = new Dock(model);
+            }
+            this.right.appendChild(this.dock.element());
+
+            // Display Buttons
+            if (!this.displayContainer) {
+                const displayContainer = new DisplayContainer();
+                const rewindDisplayIcon = new RewindDisplayIcon(model, api);
+                const playDisplayIcon = new PlayDisplayIcon(model);
+                // toggle playback
+                playDisplayIcon.on('click tap', () => {
+                    this.trigger(events.JWPLAYER_DISPLAY_CLICK);
+                    this.userActive(1000);
+                    api.play(reasonInteraction());
+                });
+                // make playDisplayIcon clickthrough on chrome for flash to avoid power safe throttle
+                if (utils.isChrome() && !utils.isMobile()) {
+                    playDisplayIcon.el.addEventListener('mousedown', () => {
+                        const provider = model.getVideo();
+                        const isFlash = (provider && provider.getName().name.indexOf('flash') === 0);
+                        if (!isFlash) {
+                            return;
+                        }
+                        const resetPointerEvents = function() {
+                            document.removeEventListener('mouseup', resetPointerEvents);
+                            playDisplayIcon.el.style.pointerEvents = 'auto';
+                        };
+                        this.style.pointerEvents = 'none';
+                        document.addEventListener('mouseup', resetPointerEvents);
+                    });
+                }
+                const nextDisplayIcon = new NextDisplayIcon(model, api);
+                displayContainer.addButton(rewindDisplayIcon);
+                displayContainer.addButton(playDisplayIcon);
+                displayContainer.addButton(nextDisplayIcon);
+                this.element.appendChild(displayContainer.element());
+                this.displayContainer = displayContainer;
+            }
+
+
+            // Display Click and Double Click Handling
+            const displayClickHandler = new ClickHandler(model, videoLayer, { useHover: true });
+            displayClickHandler.on({
+                click: () => {
+                    this.trigger(events.JWPLAYER_DISPLAY_CLICK);
```

Comment:
In this case no because `api.play` will result in events being dispatched that should come after the click event.

This is an example where DOM events with `preventDefault` could add functionality for developers who would like to override our default click behavior.